packages/components/VotingInstruction.tsx

```diff
@@ -0,0 +1,22 @@
+import { Center, Text } from '@chakra-ui/react';
+
+const VotingInstructions = () => {
+  return (
+    <Center>
+      <Text
+        fontSize="16px"
+        fontFamily="inter"
+        width="377px"
+        marginTop="96px"
```

Comment:
```suggestion
        p="6rem 1rem 1rem 1rem"
```
creio q assim fica mais responsivo e evita valores brutos. resultado aq ficou igual

Reply:
Há algum simulador que consigo ver essas equivalências? O não uso da propriedade margin é em questão há resposividade?

Reply:
acho q era interessante tirar esse width e margin em pixels mesmo a questao do padding é preferencia mesmo (como aq ta dentro de uma div nao muda muito mas tem casos q margin atrapalha o desing).

Reply:
sem o width desconfigura o componente